src/openapi/streaming/streaming.js

```diff
@@ -148,6 +148,9 @@ function onConnectionStateChanged(change) {
             break;
 
         case this.CONNECTION_STATE_RECONNECTING:
+            if (signalRTransport && signalRTransport.name === 'longPolling') {
+                log.warn(LOG_AREA, 'reconnecting using longPolling');

```

Comment:
If nothing changed as of late, We only persist by default ERROR logs. I assume this logging was mostly intended as perisisted log to check stats for fallbacks to long polling among users.

Reply:
added an isPersist flag (which the logger must handle)